src/Finch_Grid.hpp

```diff
@@ -146,17 +146,17 @@ class Grid
 
     auto getLocalGrid() { return local_grid; }
 
-    auto getIndexSpace()
+    auto getIndexSpace() const
```

Comment:
Was adding const to these functions a related change to get this to compile?

Reply:
I believe that was the case - let me check

Reply:
Removed